media/css/firefox/new/desktop/download.scss

```diff
@@ -1338,93 +1338,3 @@ button.mzp-c-cta-link {
         }
     }
 }
-
-// --------------------------------------------------------------------------
-//  Join Firefox modal
-
-.join-firefox-content {
```

Comment:
**question:** Can we remove the modal import at the top of this file?

```@import '~@mozilla-protocol/core/protocol/css/components/modal';```

Reply:
Yes! Updated.